presto-orc/src/main/java/com/facebook/presto/orc/OrcWriter.java

```diff
@@ -519,7 +519,8 @@ public void validate(OrcDataSource input)
                 input,
                 types,
                 hiveStorageTimeZone,
-                orcEncoding);
+                orcEncoding,
+                new OrcReaderConfig(false));
```

Comment:
let's don't worry about which decompressor to use for orc writer validation

Reply:
ReaderOptions will have to be supplied. Setting it to false always.